components/romio/src/ome/io/nio/PixelBuffer.java

```diff
@@ -678,14 +678,22 @@ public void setTimepoint(byte[] buffer, Integer t) throws IOException,
 
     /**
      * Retrieves the active resolution level.
-     * @return The active resolution level.
+     * @return The active resolution level.  The level will be non-negative and less
+     * than #getResolutionLevels().  Resolution level 0 is the smallest
```

Comment:
Can be updated to `{@link #getResolutionLevels()}` to improve the Javadoc markup